To: vim_dev@googlegroups.com
Subject: Patch 7.3.376
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.376
Problem:    Win32: Toolbar repainting does not work when the mouse pointer
	    hovers over a button.
Solution:   Call DefWindowProc() when not handling an event. (Sergiu Dotenco)
Files:	    src/gui_w32.c


*** ../vim-7.3.375/src/gui_w32.c	2011-08-10 17:07:56.000000000 +0200
--- src/gui_w32.c	2011-12-14 15:14:48.000000000 +0100
***************
*** 1101,1107 ****
  	return MyWindowProc(hwnd, uMsg, wParam, lParam);
      }
  
!     return 1;
  }
  
  /*
--- 1101,1107 ----
  	return MyWindowProc(hwnd, uMsg, wParam, lParam);
      }
  
!     return DefWindowProc(hwnd, uMsg, wParam, lParam);
  }
  
  /*
*** ../vim-7.3.375/src/version.c	2011-12-14 15:07:11.000000000 +0100
--- src/version.c	2011-12-14 15:23:10.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     376,
  /**/

-- 
   GALAHAD hurries to the door and pushes through it.  As he leaves the room
   we CUT TO the reverse to show that he is now in a room full of bathing
   and romping GIRLIES, all innocent, wide-eyed and beautiful.  They smile
   enchantingly at him as he tries to keep walking without being diverted by
   the lovely sights assaulting his eyeballs.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
